README.md

```diff
@@ -111,6 +111,17 @@ For example, assuming the following directory bucket name `my-test-bucket--usw2-
 usw2-az1, then the URI used will look like: `s3://my-test-bucket--usw2-az1--x-s3/<PREFIX>` (**please note that the 
 prefix for Amazon S3 Express One Zone should end with '/'**), paired with region us-west-2.
 
+## Using S3 Versioning to Manage Checkpoints
+When working with model checkpoints, users can accidentally overwrite or delete critical files. This can happen due to incorrect configuration, multiple hosts accessing the same storage path concurrently, or race conditions caused by usage of the async checkpoint interface.
+Amazon S3 bucket versioning provides a way to protect checkpoints from inadvertent loss. With versioning enabled, S3 preserves every version of every object in the bucket. You can easily recover from unintended user actions and application failures.
+When versioning is enabled on an S3 bucket:
+* S3 stores a version history of your objects. The current version is returned by default.
+* You can retrieve previous versions by specifying the version ID.
```

Comment:
I don't think "by specifying the version ID" is needed here, as the torch connector doesn't support it.